src/pages/movie/[id].js

```diff
@@ -0,0 +1,144 @@
+import Head from 'next/head';
+import {useRouter} from 'next/router';
+import styled from 'styled-components';
+
+import Layout from '../../components/Layout';
+import useFetch from '../../hooks/useFetch';
+import useStore from '../../hooks/useStore';
+
+const Wrapper = styled.section`
+	padding: 24px;
+`;
+const Picture = styled.img`
+	max-width: 185px;
+	border-radius: 16px 16px 16px 16px;
+	box-shadow: 1px 1px 10px 5px rgba(0, 0, 0, 0.2);
+`;
+
+const Hl2 = styled.h2`
+	padding: 16px 0 0 0;
+	color: white;
+	overflow-wrap: break-word;
+	font-size: 36px;
+	font-style: normal;
+	font-weight: 600;
+	& span {
+		font-weight: 400;
+	}
+`;
+
+const Hl3 = styled.h3`
+	padding: 0 8px 8px 0;
+	color: white;
+	overflow-wrap: break-word;
+	font-size: 20px;
+	font-style: italic;
+	font-weight: 400;
+`;
+
+const Hl4 = styled.h4`
+	padding: 24px 8px 0 0;
+	color: white;
+	overflow-wrap: break-word;
+	font-size: 18px;
+	font-style: normal;
+	font-weight: 600;
+	line-height: 1.3;
+	& span {
+		font-style: normal;
+		font-weight: 200;
+	}
+`;
+
+const Article = styled.article`
+	display: flex;
+	padding: 24px 0;
+`;
+
+const Paragraph = styled.p`
+	padding: 0 32px;
+	color: white;
+	line-height: 1.5;
+`;
+
+export default function Movie() {
+	// playtime helper function
+	function toHoursAndMinutes(totalMinutes) {
```

Comment:
That's at least the second time I see this function in the code. According to the rule "never repeat yourself" you could make it a utility and import it wherever needed.

Reply:
Oh, right. Great idea.
Just put it into 
 '../utils/toHoursAndMinutes';